src/admin-views/settings/tickets-commerce/stripe/connect/inactive.php

```diff
@@ -24,7 +24,7 @@
 
 <div class="tec-tickets__admin-settings-tickets-commerce-gateway-description">
 	<p>
-		<?php esc_html_e( 'Start selling tickets to your events today with Stripe. Attendees can purchase tickets directly on your site using debit or credit cards with no additional fees.', 'event-tickets' ); ?>
+		<?php echo __( 'Start selling tickets to your events today with Stripe integration for Tickets Commerce. Enable highly-configurable credit and debit card checkout with enhanced features like Afterpay.<br><br>Stripe charges a 3% fee for all transactions.', 'event-tickets' ); ?>
```

Comment:
Any reason why we're removing late escaping here? It would be useful in case po files get used as an attack vector

Reply:
Good catch. We do want to allow HTML, though. I'll add this:
```suggestion
		<?php echo wp_kses_post( __( 'Start selling tickets to your events today with Stripe integration for Tickets Commerce. Enable highly-configurable credit and debit card checkout with enhanced features like Afterpay.<br><br>Stripe charges a 3% fee for all transactions.', 'event-tickets' ) ); ?>
```